pkg/cmd/pipeline/start.go

```diff
@@ -95,49 +117,223 @@ tkn pipeline start foo --param NAME=VALUE --resource source=scaffold-git  -s Ser
 			return NameArg(args, p)
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-
-			opt := startOptions{
-				Resources:          res,
-				Params:             params,
-				ServiceAccountName: svc,
-				ServiceAccounts:    svcs,
-				Last:               last,
+			opt.stream = &cli.Stream{
+				Out: cmd.OutOrStdout(),
+				Err: cmd.OutOrStderr(),
 			}
-			return startPipeline(cmd.OutOrStdout(), opt, p, args[0])
+			return opt.run(args[0])
 		},
 	}
 
-	c.Flags().StringSliceVarP(&res, "resource", "r", []string{}, "pass the resource name and ref")
-	c.Flags().StringSliceVarP(&params, "param", "p", []string{}, "pass the param")
```

Comment:
If I understand, this PR is removing those two switches? I think that's quite a change (for at least backward compat/documentation),  there is no way to keep them ?

Reply:
We made the resource and param input interactively.

Reply:
right but you can override it with command line options, isnt it ?

Reply:
Addressed and updated the PR